distribution/docs/src/main/resources/_contents/_broker/route-manager-contents.adoc

```diff
@@ -0,0 +1,21 @@
+==== Route Manager
+
+===== Prerequisites for Route Manager
+
+None
+
+===== Installing and Uninstalling
+
+The Route Manager is installed as a part of the Broker App.
+
+===== Configuring
+
+Currently there are no configurations for the Route Manager.
+
+===== Using the Route Manager
+
+The Route Manager gives an administrator the ability to configure and deploy routes, queues, and topics dynamically. If a need arises for a new route, an administrator can easily develop a new route and deploy it to satisfy the requirement, rather than spending the time to develop, compile, and test new code.
```

Comment:
We should probably make it clear that these are Camel routes. Also might want to mention that the `sjms` component is available to routes deployed here.